hls4ml/backends/catapult/passes/recurrent_templates.py

```diff
@@ -80,17 +80,19 @@ def __init__(self):
 
     def format(self, node):
         params = self._default_config_params(node)
+        in_0, in_1 = map(str, node.get_input_variable().shape[:2])
 
-        params['n_in'] = node.get_input_variable().dim_names[1]
-        params['n_sequence'] = node.get_input_variable().dim_names[0]
+        params['n_in'] = in_1
```

Comment:
instead of the style of calling these `in_0, in_1, out_0, out_1` etc, why not give them meaningful names like they will be used later (`n_in, n_sequence, n_state` etc). Across all `*_template.py` files.